src/main/java/hibernate/tournament/controller/TournamentController.java

```diff
@@ -0,0 +1,47 @@
+package hibernate.tournament.controller;
+
+import hibernate.tournament.model.Team;
+import hibernate.tournament.repository.TeamRepository;
+import hibernate.tournament.service.TournamentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class TournamentController {
+
+    private TournamentService tournamentService;
+    private TeamRepository teamRepository;
```

Comment:
I do not recommend injecting any logic into controller.
This is not the right place for it.